e2e/test/scenarios/custom-column/cc-data-type.cy.spec.js

```diff
@@ -132,11 +139,11 @@ function addCustomColumns(columns) {
     if (index) {
       getNotebookStep("expression").icon("add").click();
     } else {
-      cy.findByText("Custom column").click();
+      cy.findByLabelText("Custom column").click();
     }
 
     enterCustomColumnDetails(column);
-    cy.button("Done").click();
+    cy.button("Done").click({ force: true });
```

Comment:
It should just work. Why do we need `force: true`?

Reply:
Because the help text popover might still be covering the button.

I could add timeouts to wait for it to go away, but I think this is more robust.

Reply:
It might be a good idea to [stress-test](https://github.com/metabase/metabase/actions/workflows/e2e-stress-test-flake-fix.yml) this change.